WtWSMS/common/culture/names_equivalency/00_names.txt

```diff
@@ -2018,13 +1982,11 @@ hval_male = { Hval }
 hvaredhay_female = { Hvaredhay }
 hvovi_female = { Hvovi }
 hyacinth_male = { Giacinto Hyacinth Iaquintus Jacek Jacenty }
-hyeronimus_male = { Gerolamo Geronimo Girolamo ZirO_nimu }
 hygia_female = { Hygia }
-hyginus_male = { Igino Iginu }
+hyginos_male = { Hyginos Hyginus Igino Iginu }
 hypatia_female = { Hypatia }
-hypatios_male = { Epate Hypatios }
+hypatios_male = { Epate Hypatios Hypatius }
 hyperechia_female = { Epyrekhia Hyperechia }
-hyppolite_male = { Ippolito PO_litu }
```

Comment:
?

Reply:
> 

Duplicate equivalency, names moved to hippolytos_male.